eva-pipeline/src/test/java/embl/ebi/variation/eva/pipeline/jobs/VariantConfigurationTest.java

```diff
@@ -133,66 +133,51 @@ public void validTransform() throws JobExecutionException, IOException {
     @Test
     public void invalidTransform() throws JobExecutionException {
         String input = VariantConfigurationTest.class.getResource(FILE_WRONG_NO_ALT).getFile();
-        String opencgaHome = System.getenv("OPENCGA_HOME") != null ? System.getenv("OPENCGA_HOME") : "/opt/opencga";
         String dbName = INVALID_TRANSFORM;
 
-        JobParameters parameters = new JobParametersBuilder()
-                .addString("input", input)
-                .addString("outputDir", "/tmp")
-                .addString("dbName", dbName)
-                .addString("compressExtension", ".gz")
-                .addString("compressGenotypes", "true")
-                .addString("includeSrc", "FIRST_8_COLUMNS")
-                .addString("aggregated", "NONE")
-                .addString("studyType", "COLLECTION")
-                .addString("studyName", "studyName")
-                .addString("studyId", "2")
-                .addString("fileId", "2")
-                .addString("opencga.app.home", opencgaHome)
-                .addString(VariantsLoad.SKIP_LOAD, "true")
-                .addString(VariantsStatsCreate.SKIP_STATS_CREATE, "true")
-                .addString(VariantsStatsLoad.SKIP_STATS_LOAD, "true")
-                .addString(VariantsAnnotGenerateInput.SKIP_ANNOT_GENERATE_INPUT, "true")
-                .addString(VariantsAnnotCreate.SKIP_ANNOT_CREATE, "true")
-                .addString(VariantsAnnotLoad.SKIP_ANNOT_LOAD, "true")
-                .toJobParameters();
-
-        JobExecution execution = jobLauncher.run(job, parameters);
-
-        assertEquals(input, execution.getJobParameters().getString("input"));
+        pipelineOptions.put("input", input);
+        variantOptions.put(VariantStorageManager.DB_NAME, dbName);
+
+        VariantSource source = (VariantSource) variantOptions.get(VariantStorageManager.VARIANT_SOURCE);
+
+        variantOptions.put(VariantStorageManager.VARIANT_SOURCE, new VariantSource(
+                input,
+                source.getFileId(),
+                source.getStudyId(),
+                source.getStudyName(),
+                source.getType(),
+                source.getAggregation()));
+
+        JobExecution execution = jobLauncher.run(job, getJobParameters());
+
+        assertEquals(input, pipelineOptions.getString("input"));
         assertEquals(ExitStatus.FAILED.getExitCode(), execution.getExitStatus().getExitCode());
     }
 
-    @Test
+/*    @Test
     public void validLoad() throws JobExecutionException, IllegalAccessException, ClassNotFoundException,
             InstantiationException, IOException, StorageManagerException {
```

Comment:
I've removed the comment and leave the tests to run.. I also added a todo comment and I will review this in https://www.ebi.ac.uk/ena/jira/browse/EVA-387